src/components/TestResult/ResultErrorBody.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+MIT License
+
+Copyright (c) 2021-present, Elastic NV
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+import { EuiFlexGroup, EuiFlexItem, EuiText, EuiCodeBlock } from "@elastic/eui";
+import React from "react";
+import { ResultCategory } from "../../common/types";
+import { ResultErrorAccordion, symbols } from "./styles";
+
+function removeColorCodes(str = "") {
+  // eslint-disable-next-line no-control-regex
+  return str.replace(/\u001b\[.*?m/g, "");
```

Comment:
How confident are we that this RegEx always catches color codes? Can't we disable the color codes when we invoke the process which yields the error message instead? That feels like it would be more reliable and less work.

I've heard a funny quote in a talk once:

> I had a problem. I tried to use a RegEx. I then had two problems.

😆

Reply:
Agree, not crazy about it. This code isn't new to this PR though, just moved from an old one.

Want me to simply remove this and render the raw value?